doc/concepts/configuration/configuration_memtx.rst

```diff
@@ -0,0 +1,233 @@
+..  _configuration_memtx:
+
+In-memory storage
+=================
+
+**Example on GitHub**: `memtx <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/config/instances.enabled/memtx>`_
+
+In Tarantool, all data is stored in random-access memory (RAM) by default.
+For this purpose, the :ref:`memtx <engines-memtx>` storage engine is used.
+
+This topic describes how to define settings related to in-memory storage in the
+:ref:`memtx <configuration_reference_memtx>` section of a :ref:`YAML configuration <configuration>`.
+
+..  NOTE::
+
+    To calculate the required amount of memory, you can use the
+    `sizing calculator <https://www.tarantool.io/en/sizing_calculator/>`_.
+
+..  _configuration_memtx-memory:
+
+Configure the memory size
+-------------------------
+
+In Tarantool, data is stored in spaces.
+Each space consists of tuples -- the database records.
+To specify the amount of memory that Tarantool allocates to actually store tuples, use the
+:ref:`memtx.memory <configuration_reference_snapshot_dir>` configuration option.
+
+In the example below, the memory size is 100000000 bytes, which is about 95 MB.
+The server does not exceeds this limit to allocate tuples.
+For indexes and connection information, the additional memory is used.
+
+When the ``memtx.memory`` limit is reached, ``INSERT`` or ``UPDATE`` requests fail with
+:ref:`ER_MEMORY_ISSUE <admin-troubleshoot-memory-issues>`.
+
+..  literalinclude:: /code_snippets/snippets/config/instances.enabled/memtx/config.yaml
+    :language: yaml
+    :start-at: memtx:
+    :end-at: 100000000
+    :dedent:
+
+..  _configuration_memtx-allocator:
+
+Specify the allocator type
+--------------------------
+
+The allocators are the memory managers used for different purposes.
+To set the allocator type that manages memory for ``memtx`` tuples, use the :ref:`memtx.allocator <configuration_reference_memtx_allocator>`
+configuration option.
+
+Possible types:
+
+*   ``system`` -- the memory is allocated as needed, checking that the quota is not exceeded.
+
+*   ``small`` (default) -- a `slab allocator <https://github.com/tarantool/small>`_ mechanism is used.
+
+The example below shows how to specify the ``small`` allocator type:
+
+..  literalinclude:: /code_snippets/snippets/config/instances.enabled/memtx/config.yaml
+    :language: yaml
+    :start-at: allocator:
+    :end-at: 'small'
+    :dedent:
+
+..  _configuration_memtx-min-max:
+
+Specify the size limitations of the allocation unit
+---------------------------------------------------
+
+You can configure the sizes of the smallest and the largest units to allocate.
+
+To set the minimum size in bytes, use the :ref:`memtx.min_tuple_size <configuration_reference_memtx_min_size>` configuration option.
+The setting can be useful if the tuples that you store are small.
+The minimum size is a value between 8 and 1048280 inclusive.
```

Comment:
Let's keep such reference info in the configuration reference only. It might be hard to maintain such info in several places.